### PR TITLE
New Post: Plugins

### PR DESCRIPTION
**Pull request by MrTronMan**

### CODE REVIEW

**Comment by KorbsStudio** [comment]
This shouldn't start out with:
```
<h1 id="getting-started-with-plugins">Getting started with plugins</h1>
```
Since the title belongs in the sidebar of the article, although I liked that you used an ID for links and such.

Also, other areas of the sidebar need to be edited (Report, Edit, etc).

Also, I think we can more informational, like fully explaining why users shouldn't have the server running while installing new plugins. Also, explain why users should restart instead of reloading(as this can break plugins like DiscordSRV).
I want our help center to provide as much information as possible, even if it's a lot! 

Also during this update of our help center, we'll like to correct our grammar, I have been using Grammarly for this:
image

**Comment by MrTronMan** [comment]
I have renamed it to plugins instead of installing plugins because this is all about plugins now 😄 

I have added more detail such as a way to turn off the plugin without deleting it. I have also added details on why you should turn off the server and also not reload, I have also moved that more toward the top.

**Review by KorbsStudio — CHANGES_REQUESTED** [review]
For **Turning off a plugin** and **Turning back on the plugin**, they should both be changed to `<h3>` instead of `<h2>` and should be put under a `<h2>` title called **Enabling and Disabling Plugins**. Of course, changing both of the `<h3>` titles to **Enabling** and **Disabling**.

Example:
# Enabling and Disabling
## Enabling
## Disabling
__ 
Add explanation behind what a YML file is for **Configuration**.

A list of recommendations for plugins might be added in the future, providing alternatives.

**Comment by MrTronMan** [comment]
**Updated the plugins.html**

Added, what you said. Added an explanation on `YMLs` (YAMLS). Also decided to rename some of the headings.

**Comment by KylePlays-Studio** [comment]
this falix trash yeet polarnodes better than falixnodes yeet

**Review by MrTronMan — COMMENTED** [review]
Updated the plugins.html

Added, what you said. Added an explanation on YMLs (YAMLS). Also decided to rename some of the headings.

File: article/minecraft/general/plugins/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          <div class="sidebar">
+            <!-- <div class="youtube">
+                <h3 style="background: #ff4141;">YouTube Tutorial</h3>
+                <iframe id="youtube_embed" width="260px" height="148px" src="https://www.youtube-nocookie.com/embed/BQe1osbKuKQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            </div> -->
+            <div class="meta">
+                <h3>Meta</h3>
+                <p id="title">Plugins</p>
+                <p id="date">Last updated: March 25th, 2021</p>
+                <p id="category">Minecraft General/p>
```
Comment (KorbsStudio) [inline article/minecraft/general/plugins/index.html:24]: Syntax error

**Review by MrTronMan — COMMENTED** [review]
Moved file locations, as well as I, fixed the syntax error.

**Comment by KylePlays-Studio** [comment]
stfu dis surver suks

**Comment by KylePlays-Studio** [comment]
stfu dis surver suks

**Comment by KylePlays-Studio** [comment]
stfu dis surver suks

**Comment by KylePlays-Studio** [comment]
stfu dis surver suks

**Comment by KylePlays-Studio** [comment]
stfu dis surver suks

**Comment by KylePlays-Studio** [comment]
stfu dis surver suks

**Comment by MrTronMan** [comment]
Added more about the plugin server jars, where to get plugin server jars.

**Comment by MrTronMan** [comment]
You did a revamp and changed it to Markdown so I will convert and update the repository.

**Pull request closed without merging**